3.6.1/Dockerfile

```diff
@@ -0,0 +1,137 @@
+#
+# Dockerfile for Python
+#
+FROM ubuntu:16.04
+MAINTAINER Azure App Service Container Images <appsvc-images@microsoft.com>
+
+
+# ========
+# ENV vars
+# ========
+
+#python
+ENV PYTHON_VERSION "3.6.1"
+ENV PYTHON_TAR_URL "https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tar.xz"
+ENV PYTHON_TAR_MD5 "692b4fc3a2ba0d54d1495d4ead5b0b5c"
+# nginx
+ENV NGINX_LOG_DIR "/home/LogFiles/nginx"
+# uwsgi
+ENV UWSGI_INI_DIR "/home/uwsgi"
+# ssh
+ENV SSH_PASSWD "root:Docker!"
+#
+ENV DOCKER_BUILD_HOME "/dockerbuild"
+
+
+# =======
+# Install
+# =======
+
+WORKDIR $DOCKER_BUILD_HOME
+
+RUN set -ex \
+	# tools
+	&& tools=' \
+		gcc \
+		make \
+		wget \
+	' \
+	&& apt-get update \
+        && apt-get install -y -V --no-install-recommends $tools \
+        && rm -r /var/lib/apt/lists/* \
+	
+	# build time libs
+	&& buildTimeDeps=' \
+		dpkg-dev \
+		libbz2-dev \
+		libc6-dev \
+		libexpat1-dev \
+		libffi-dev \
+		libgdbm-dev \
+		liblzma-dev \
+		libncurses-dev \
+		libreadline-dev \
+		libsqlite3-dev \
+		libssl-dev \
+		tcl-dev \
+		tk-dev \
+		xz-utils \
+		zlib1g-dev \
+	' \
+	&& apt-get update \
+	&& apt-get install -y -V --no-install-recommends $buildTimeDeps \
+	&& rm -r /var/lib/apt/lists/*
+
+RUN set -ex \
+	#python
+	&& cd $DOCKER_BUILD_HOME \
+	&& wget -O python.tar.xz "$PYTHON_TAR_URL" --no-check-certificate \
+	&& echo "$PYTHON_TAR_MD5 *python.tar.xz" | md5sum -c - \
+	&& mkdir -p /usr/src/python \
+	&& tar -xf python.tar.xz -C /usr/src/python --strip-components=1 \
+	&& cd /usr/src/python \
+	&& gnuArch="$(dpkg-architecture --query DEB_BUILD_GNU_TYPE)" \
+	&& ./configure \
+		--build="$gnuArch" \
+		--enable-loadable-sqlite-extensions \
+		--enable-shared \
+		--with-system-expat \
+		--with-system-ffi \
+		--without-ensurepip \
+	&& make -j "$(nproc)" \
+	&& make install \
+	&& ldconfig \
+	
+	# pip
+	&& wget https://bootstrap.pypa.io/get-pip.py --no-check-certificate \
```

Comment:
now apt-get install pip with version 8.1.1, furthermore, [get-pip.py](https://github.com/pypa/get-pip) install the latest version of pip